src/Data/Fin/Properties.agda

```diff
@@ -131,6 +131,13 @@ fromℕ≤≡fromℕ≤″ (s≤s z≤n)       (ℕ.less-than-or-equal refl) = r
 fromℕ≤≡fromℕ≤″ (s≤s (s≤s m<n)) (ℕ.less-than-or-equal refl) =
   cong suc (fromℕ≤≡fromℕ≤″ (s≤s m<n) (ℕ.less-than-or-equal refl))
 
+toℕ-fromℕ≤″ : ∀ {m n} (m<n : m ℕ.<″ n) → toℕ (fromℕ≤″ m m<n) ≡ m
+toℕ-fromℕ≤″ {m} {n} m<n = begin
+  toℕ (fromℕ≤″ m m<n)  ≡⟨ cong toℕ (sym (fromℕ≤≡fromℕ≤″ _ m<n)) ⟩
+  toℕ (fromℕ≤ _)       ≡⟨ toℕ-fromℕ≤ (ℕₚ.≤″⇒≤ m<n) ⟩
+  m ∎
+  where open P.≡-Reasoning
+
```

Comment:
Cosmetic nitpick: these days I quite like `let open P.≡-Reasoning in begin (...)` on the
top line (or rather `let open RawMonad m in do (...)` in my case) because it makes the kit
we are using explicit for the reader.

Reply:
Hmm, we use the "where" syntax throughout the library. It would be quite an undertaking to change it all. I feel that given it is cosmetic, maybe better to stick with being consistent for the moment.